common/helper_test.go

```diff
@@ -368,10 +343,120 @@ func TestHelpersValueFromRemoteObject(t *testing.T) {
 				Value: marshalled,
 			}
 
-			arg, err := valueFromRemoteObject(ctx, remoteObject)
+			logger := NewLogger(ctx, logrus.New(), false, nil)
+			arg, err := valueFromRemoteObject(ctx, remoteObject, logger.logger.WithContext(ctx))
 
 			require.Nil(t, err)
 			require.Equal(t, p.value, p.toFn(arg))
 		}
 	})
+
+	t.Run("remote object with ID", func(t *testing.T) {
+		rt := goja.New()
+		ctx := k6common.WithRuntime(context.Background(), rt)
```

Comment:
`rt` is used on line 371.

In general I prefer initializing these commonly used variables up front, even if they're only used once, since it makes it easier to expand the test later.